Display Issue in gauge-vscode v0.1.4: Absence of 'Run Scenario' and 'Debug Scenario' Labels

### Describe the bug
gauge-vscode v0.1.4 is not working correctly because the labels "Run Scenario" and "Debug Scenario" no longer appear. I have tried several times, but nothing is displayed. However, once I reinstall the old version gauge-vscode v0.1.0, everything works correctly! (see associated screenshots)

<img width="853" alt="KO-v0 1 4" src="https://github.com/getgauge/gauge-vscode/assets/144631369/6c6ea087-946b-453c-860d-9b9e3fbb63ca">
<img width="885" alt="OK-v0 1 0" src="https://github.com/getgauge/gauge-vscode/assets/144631369/15a51fda-3424-4fce-b68f-d988566c1f48">

#### Gauge (Output of `gauge -v`)
Gauge version: 1.5.6
Commit Hash: 9b96118

Plugins
-------
confluence (0.18.4)
dotnet (0.5.2)
html-report (4.2.0)
inprogress (0.4.0)
java (0.10.3)

#### Java version 
"11.0.15" 2022-04-19 LTS#### Operating System information

Looking at https://github.com/getgauge/gauge/issues/2431 it seems like windows users are affected.
I gather from this logline
`13-11-2023 11:16:51.791 [python] [ERROR] '"C:\Program Files\Python310\python.exe -m pip install getgauge==0.4.1 --user"' is not recognized as an internal or external command,`
that

1. VScode is starting vscode-gauge
2. vscode-gauge starts gauge
3. gauge correctly identifies the language plugin to use
4. gauge starts [start.sh](https://github.com/getgauge/gauge-python/blob/master/start.bat) 
5. start.bat starts [check_and_install_getgauge.py](https://github.com/getgauge/gauge-python/blob/master/check_and_install_getgauge.py)
6. there it fails because it cannot resolve(?) the command

This feels like a path issue to me, even if I don't have a clue how it could've been introduced.

@EGTomKies It may be interesting to output the available path in `start.bat` or `check_and_install_getgauge.py`, to see if this is the case.
The plugins are located in the home directory at `.gauge/plugins`. Editing `start.bat` there should take effect directly.

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   "description": "Gauge support for VScode.",
   "author": "ThoughtWorks",
   "license": "MIT",
-  "version": "0.1.4",
+  "version": "0.1.5",
   "publisher": "getgauge",
   "engines": {
     "vscode": "^1.71.0"
@@ -480,7 +480,7 @@
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
-    "clipboardy": "^4.0.0",
+    "clipboardy": "^2.3.0",
     "fs-extra": "^11.1.1",
     "get-port": "^5.1.1",
     "ps-tree": "^1.2.0",

--- a/src/annotator/generateStub.ts
+++ b/src/annotator/generateStub.ts
@@ -9,7 +9,7 @@ import { ProjectFactory } from "../project/projectFactory";
 import { WorkspaceEditor } from "../refactor/workspaceEditor";
 import { FileListItem } from "../types/fileListItem";
 
-import clipboardy from 'clipboardy';
+import clipboardy = require("clipboardy");
 
 export class GenerateStubCommandProvider implements Disposable {
     private readonly _clientsMap: GaugeClients;

